ApprovalTests/reporters/DiffPrograms.h

```diff
@@ -124,6 +160,13 @@ namespace ApprovalTests
                 DiffInfo("{ProgramFiles}Devart\\Code Compare\\CodeCompare.exe",
                          Type::TEXT))
 
+            APPROVAL_TESTS_MACROS_ENTRY(
+                SUBLIME_MERGE,
+                DiffInfo(
+                    "{ProgramFiles}Sublime Merge\\\\smerge.exe",
```

Comment:
A small thing: I see that there are 4 backslashes on this line, and 2 on the others... Perhaps an over-helpful IDE?

I'm happy to fix this after merging, but thought I'd just check first...

Reply:
I want to blame CMake for ruining me 😅, but I think this was just the IDE this time. Fixed.